util/pkg/vfs/s3fs.go

```diff
@@ -61,6 +61,14 @@ func newS3Path(s3Context *S3Context, bucket string, key string) *S3Path {
 }
 
 func (p *S3Path) Path() string {
+	response,_ := p.GetVersioning()
+	if response != true {
+		fmt.Println("S3 Bucket Versioning is not set. Let's set it")
+		p.SetVersioning("Enabled")
```

Comment:
I do not think we can enable versioning for everyone.  Let's just log that it is not on.

Reply:
done